packages/code-du-travail-frontend/src/common/Answer.js

```diff
@@ -62,15 +60,29 @@ function Answer({
       )
     ).reduce((state, node) => {
       glossary.forEach(item => {
+        // we cannot use \b word boundary since \w does not match diacritics
+        // So we do a kind of \b equivalent.
+        // the main différence is that matched pattern can include a whitespace as first char
+        const frDiacritics = "àâäçéèêëïîôöùûüÿœæÀÂÄÇÉÈÊËÎÏÔÖÙÛÜŸŒÆ";
+        const wordBoundaryStart = `(?:^|[^\\w${frDiacritics}])`;
+        const wordBoundaryEnd = `(?![\\w${frDiacritics}])`;
         const patterns = [...new Set([item.title, ...item.variants])]
```

Comment:
shouldn't we extract the definition of patterns this from the loop ? . maybe a top-level fonction

Reply:
Not sure why we should do this ? 
Would you prefer I build a map of each term associated to its definiton, loop over it to made the html replacement and then get all the nodes before calling portals ?

Reply:
looks like the patterns list is recreated on each iteration ?

Reply:
oh I get it ! thanks